app/views/layouts/base.html.erb

```diff
@@ -54,9 +54,10 @@
 </head>
 
 <body class="site <%= yield(:background_cls) %>">
+<%= link_to t('shared.skip_link'), '#main-content', class: 'usa-skipnav' %>
   <div class="usa-overlay"></div>
   <%= yield(:mobile_nav) if content_for?(:mobile_nav) %>
-  <div class="site-wrap bg-light-blue">
+  <div class="bg-light-blue">
```

Comment:
Do we need this wrapper element anymore?

Reply:
We do not. Originally, it was there to ensure a background was on the `/account` page, but since the color is now on the `main` element, it can be removed.